pkg/imgpkg/cmd/experimental_flags.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import "github.com/spf13/cobra"
+
+type ExperimentalFlags struct {
+	RecursiveBundles bool
+}
+
+func (s *ExperimentalFlags) Set(cmd *cobra.Command) {
+	cmd.Flags().BoolVar(&s.RecursiveBundles, "experimental-recursive-bundle", false, "Enable the experimental functionality of Bundles of Bundles")
```

Comment:
do we need a flag for this functionality?

Reply:
we do not need a flag for the functionality, but if we want to be able to keep on releasing `imgpkg` while we are building the full epic, it is better if we hide it behind this flag.
There will be one last story that will completely remove this flag